ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml

```diff
@@ -73,6 +73,18 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-ibmcloud-ipi-ovn
   timeout: 5h0m0s
+- as: data-path-9nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: ibmcloud-qe
```

Comment:
Will this be a periodic job?

Reply:
If so, please add the cron job

Reply:
@qiliRedHat  Please let know if I will have to make this periodic

Reply:
@SachinNinganure No for IBM Cloud we don't run cron now.

Reply:
@qiliRedHat - thank you!
@SachinNinganure: Please add `always_run: false`. For reference [URL](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml#L103)